src/EditPopup.js

```diff
@@ -0,0 +1,32 @@
+import React, {useCallback, useState} from 'react';
+import './Popup.css';
+
+
+const EditPopup = ({show, closeEdit, onSubmit}) => {
+    const [value, setValue] = useState("");
+
+    const handleChange = useCallback(event => setValue(event.target.value), []);
+    const handleSubmit = useCallback((e) => {
+        e.preventDefault();
+        onSubmit(value);
+        setValue("");
+        closeEdit();
+    }, [value, onSubmit, closeEdit]);
+
+    return(
+        <div className={"editWrapper" + (show ? '' : ' hidden')}>
+            <div className={'edit'} id="editPopup">
+                <h1 className="editHeader">Would you like to change your Idea?</h1>
```

Comment:
Although names are good, you don't need to always use a class to target an element with CSS. In this case, your selector can be `.edit h1` and it'll still be unique enough.